apiary.apib

```diff
@@ -8280,6 +8281,37 @@ Create a ticket.
             + type: `ticket` (string)
             + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
 
+### tickets.update [POST /tickets.update]
+
+Update a ticket.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, required)
+        + subject: `My ticket subject` (string, optional)
+        + description: `My ticket details` (string, optional) - Uses Markdown formatting
+        + ticket_status_id: `46156648-87c6-478d-8aa7-1dc3a00dacab` (string, optional)
+        + customer (object, optional)
+            + type: `company` (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, required)
+        + assignee (object, optional, nullable)
+            + type (enum[string], required)
+                + Members
+                    + `user`
+            + id: `98b2863e-7b01-4232-82f5-ede1f0b9db22` (string, required)
+        + participant (object, optional, nullable)
+            + customer (object, required, nullable)
+                + type (enum, required)
+                    + Members
+                        + `company`
```

Comment:
Q: Aren't we missing `contact` here?

Reply:
Currently, referencing a customer participating in a ticket is limited to companies only.